intelmq/bots/experts/taxonomy/expert.py

```diff
@@ -44,24 +45,25 @@
     "unauthorized-command": "intrusions",  # not in ENISA eCSIRT-II taxonomy
     "unauthorized-login": "intrusions",  # not in ENISA eCSIRT-II taxonomy
     "unprivileged-account-compromise": "intrusions",
-    "c2server": "malicious code",  # ENISA eCSIRT-II taxonomy: 'c2server'
-    "dga domain": "malicious code",  # not in ENISA eCSIRT-II taxonomy
-    "infected-system": "malicious code",  # ENISA eCSIRT-II taxonomy: 'infected-system'
-    "malware": "malicious code",  # not in ENISA eCSIRT-II taxonomy
-    "malware-configuration": "malicious code",  # ENISA eCSIRT-II taxonomy: 'malware-configuration'
-    "malware-distribution": "malicious code",
-    "ransomware": "malicious code",  # not in ENISA eCSIRT-II taxonomy
-    "blacklist": "other",
-    "other": "other",  # not in ENISA eCSIRT-II taxonomy
+    "c2-server": "malicious-code",
+    "dga-domain": "malicious-code",  # not in ENISA eCSIRT-II taxonomy
+    "infected-system": "malicious-code",
+    "malware": "malicious-code",  # not in ENISA eCSIRT-II taxonomy
+    "malware-configuration": "malicious-code",
+    "malware-distribution": "malicious-code",
+    "ransomware": "malicious-code",  # not in ENISA eCSIRT-II taxonomy
+    "blacklist": "other",  # not in ENISA eCSIRT-II taxonomy
+    "other": "other",
+    "undetermined": "other",
     "proxy": "other",  # not in ENISA eCSIRT-II taxonomy
     "tor": "other",  # not in ENISA eCSIRT-II taxonomy
     "unknown": "other",  # not in ENISA eCSIRT-II taxonomy
     "test": "test",
     "ddos-amplifier": "vulnerable",
     "information-disclosure": "vulnerable",
     "potentially-unwanted-accessible": "vulnerable",
-    "vulnerable client": "vulnerable",  # not in ENISA eCSIRT-II taxonomy
-    "vulnerable service": "vulnerable",  # not in ENISA eCSIRT-II taxonomy
+    "vulnerable-client": "vulnerable",  # not in ENISA eCSIRT-II taxonomy
+    "vulnerable-service": "vulnerable",  # not in ENISA eCSIRT-II taxonomy
```

Comment:
The correct RSIT type is actually `vulnerable-system` for both `vulnerable client` and `vulnerable service`